org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/validation/ValidatorFragment2.xtend

```diff
@@ -232,6 +245,7 @@ class ValidatorFragment2 extends AbstractInheritingFragment {
 			
 				@Override
 				protected void initialize(�IAcceptor�<�PreferenceKey�> acceptor) {
+					super.initialize(acceptor);
```

Comment:
so we always call super.initialize(acceptor) even for non Xbase languages, is that right?
This sounds sensible, but just to be sure: in the current version for super.initialize was never called

Reply:
Yes. In the current version this is not done since it wasn't necessary. Doing now for all langs does not hurt.